Makefile

```diff
@@ -222,18 +222,9 @@ dev: ## Set Environment to DEV
 	$(eval export DISABLED_AZS)
 	@true
 
-.PHONY: dev01
-dev01: dev
-	$(eval export DEPLOY_ENV=dev01)
-	$(eval export ENABLE_AUTODELETE=false)
-	$(eval export ENABLE_AZ_HEALTHCHECK ?= false)
-	$(eval export SYSTEM_DNS_ZONE_NAME=${DEPLOY_ENV}.dev.cloudpipeline.digital)
-	$(eval export APPS_DNS_ZONE_NAME=${DEPLOY_ENV}.dev.cloudpipelineapps.digital)
-	@true
-
-.PHONY: dev02
-dev02: dev
-	$(eval export DEPLOY_ENV=dev02)
+.PHONY: $(filter-out dev%,$(MAKECMDGOALS))
+dev%: dev
+	$(eval export DEPLOY_ENV=$@)
```

Comment:
We need to include those extra lines because they otherwise don't include the new deploy env name. `eval export` does it at the time it's invoked, with the context it's invoked in.

Reply:
I misread the diff